docs/defined_scientific.rst

```diff
@@ -45,7 +46,17 @@ SREX Regions
 
     @savefig plotting_srex.png width=100%
     plt.tight_layout()
-    
+
+CPC Regions
+============
+
+.. ipython:: python
+
+    regionmask.defined_regions.climate_indices.plot();
```

Comment:
```suggestion
    regionmask.defined_regions.climate_indices.plot(label="abbrev");
```
If you switch to one `Regions` instance per index you could combine some in to one plot.